### PR TITLE
Updated buildscripts and spec files

### PR DESCRIPTION
**Pull request by aazam-gh**
Updated spec files for `rhel` and `sles` for `zthin-parts` and their buildscripts.

### CODE REVIEW

File: buildzthinrpm_rhel [hunk 1]
```diff
@@ -26,15 +26,15 @@ fi
...
+REPO=https://github.com/openmainframeproject/feilong
+REPOTOP=feilong
 ZTHINTOP=zthin-parts
 
 # Check whether branch exists or not
 EXISTS=$(git ls-remote -h $REPO | grep "$BRANCH"$ | wc -l )
 if [ "$EXISTS" == 0 ] ; then
    {
-   echo "Branch $BRANCH does not exist on /mfcloud/python-zvm-sdk"
+   echo "Branch $BRANCH does not exist on /openmainframeproject/feilong"
```
Comment (bjhuangr) [inline buildzthinrpm_rhel]: echo "Branch $BRANCH does not exist on openmainframeproject/feilong"

File: buildzthinrpm_sles [hunk 2]
```diff
@@ -24,15 +24,15 @@ fi
...
+REPO=https://github.com/openmainframeproject/feilong
+REPOTOP=feilong
 ZTHINTOP=zthin-parts
 # SMCLIOPTDIR=/opt/zthin/bin/smcli
 
 EXISTS=$(git ls-remote -h $REPO | grep "$BRANCH"$ | wc -l )
 if [ "$EXISTS" == 0 ] ; then
    {
-   echo "Branch $BRANCH does not exist on /mfcloud/python-zvm-sdk"
+   echo "Branch $BRANCH does not exist on /openmainframeproject/feilong"
```
Comment (bjhuangr) [inline buildzthinrpm_sles]: ditto

File: buildzthinrpm_rhel (hunk 1 above)
Comment (aazam-gh) [reply buildzthinrpm_rhel]: Done

File: buildzthinrpm_sles (hunk 2 above)
Comment (aazam-gh) [reply buildzthinrpm_sles]: Done

File: buildzthinrpm_rhel (hunk 1 above)
Comment (bjhuangr) [reply buildzthinrpm_rhel]: I meant, remove the `/` before `openmainframeproject/feilong`

Comment (Bischoff) [reply buildzthinrpm_rhel]: seems done now

**Comment by bjhuangr** [comment]
@aazam-gh still WIP?

**Comment by aazam-gh** [comment]
@bjhuangr PR is ready to be merged. Current changes will suffice.

**Review by bjhuangr — APPROVED** [review]

**Pull request merged**